great_expectations/util.py

```diff
@@ -1069,3 +1070,11 @@ def generate_library_json_from_registered_expectations():
 
 def delete_blank_lines(text: str) -> str:
     return re.sub(r"\n\s*\n", "\n", text, flags=re.MULTILINE)
+
+
+def generate_temporary_table_name(
+    default_table_name_prefix: Optional[str] = "ge_tmp_",
+    num_digits: Optional[int] = 8,
+) -> str:
+    table_name: str = f"{default_table_name_prefix}{str(uuid.uuid4())[:num_digits]}"
```

Comment:
What about dialect-specific cases such as MSSQL requiring temp tables to start with `#`?

Reply:
@NathanFarmer One can override the default prefix.  Let me fix this.  For some reason "grep" missed this one.  Nice catch.